examples/main_profile_feature.go

```diff
@@ -0,0 +1,79 @@
+// to run the CPU profiling: go build -ldflags "-X main.RunCPUProfile=true" main.go && ./main
+// to run the Mem profiling: go build -ldflags "-X main.RunMemProfile=true" main.go && ./main
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/optimizely/go-sdk/optimizely/client"
+	"github.com/optimizely/go-sdk/optimizely/decision"
+	"github.com/optimizely/go-sdk/optimizely/entities"
+	"github.com/optimizely/go-sdk/optimizely/notification"
+
+	"github.com/pkg/profile"
+)
+
+// stressTest has everything that test app has. it is used to run profile
+func stressTest() {
+	/*
+		For the test app, the biggest json file is used with 100 entities.
+		DATAFILES_DIR has to be set to point to the path where 100_entities.json is located.
+	*/
+
+	var datafileDir = path.Join(os.Getenv("DATAFILES_DIR"), "100_entities.json")
+
+	datafile, err := ioutil.ReadFile(datafileDir)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
should we panic to bail out of the prof?

Reply:
does not matter, but I will remove it